app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -41,6 +41,10 @@ def supports_authentication?(authtype)
       supported_auth_types.include?(authtype.to_s)
     end
 
+    def supported_catalog_types
+      %w(vmware)
```

Comment:
Are any orchestration templates supported by vSphere?

Reply:
@agrare 
Orchestration templates are not supported for vSphere.

Reply:
So should this be left as `[]` ?

Reply:
@agrare This is not for orchestration templates alone, this is for service that provision a vm.

Reply:
Ah okay didn't realize that thanks